### ISSUE TITLE
Angular v1.4.x

### ISSUE THREAD

**Issue by lvanderbijl** [angular-ui/angular-google-maps#1352]
Does angular google maps work with v1.4.x versions of Angular?

Judging by a recent reversion of the bower file, I'm guessing it doesn't at the moment? 

Thanks for your time

**Comment by nmccready** [comment]
It may, I have not run against it thoroughly. My main worry is if they changed anything to the underlying structure of promises. There are some hacks going on to check the promise state so we can kill them early if need be. I noticed this changed from 1.2 to 1.3. My bet is it may change again in 1.4.

These are the lines of concern. 
https://github.com/angular-ui/angular-google-maps/blob/master/src/coffee/directives/api/utils/promise.coffee#L23-L30

Basically if someone wants to give it a go run the exampl.html  and ./example/issue-979-polygons-race.html

**Comment by nikshulipa** [comment]
+1 for support v1.4.x

**Comment by dingels35** [comment]
:+1:

**Comment by cthrax** [comment]
So what doesn't work? Someone just needs to run the example, as stated above.

**Comment by wrummler** [comment]
I ran that example against ng 1.4.1, and it seemed to repeatedly clear and make those polygons like a champ. No errors logged, other log messages piping out seemingly normally.

Since 1.4.x was released, I've been using and testing it against agm 2.1.x in an app I've been working on. The app's not in production yet, but so far I've experienced zero problems with ng 1.4.1 and agm.

**Comment by nmccready** [comment]
Cool ill try bumping it.  Thanks for the look.

**Comment by nmccready** [comment]
Yeah looks good to me, ill push it.

**Comment by nmccready** [comment]
Done one releases:

- [2.1.5](../../releases/tag/2.1.5)
- [2.0.23](../../releases/tag/2.0.23)
